packages/paste-website/src/pages/patterns/pricing/index.mdx

```diff
@@ -0,0 +1,390 @@
+          <Td>
+            6% of monthly spend <DetailText marginTop="space0">or $1500.00 minimum</DetailText>
+          </Td>
+          <Td>
+            8% of monthly spend <DetailText marginTop="space0">or $5000.00 minimum</DetailText>
+          </Td>
+        </Tr>
+        <Tr>
+          <Td>Web support</Td>
+          <Td>
+            <SelectedIcon decorative color="colorTextIconAvailable" />
+          </Td>
+          <Td>
+            <SelectedIcon decorative color="colorTextIconAvailable" />
+          </Td>
+          <Td>
+            <SelectedIcon decorative color="colorTextIconAvailable" />
+          </Td>
+          <Td>
+            <SelectedIcon decorative color="colorTextIconAvailable" />
+          </Td>
+        </Tr>
+        <Tr>
+          <Td>
+            Guaranteed response times <DetailText marginTop="space0">Business critical</DetailText>
+          </Td>
+          <Td>-</Td>
+          <Td>3 business hours</Td>
+          <Td>1 hour (24x7)</Td>
+          <Td>1 hour (24x7)</Td>
+        </Tr>
+        {...}
+      </TBody>
+    </Table>
+  )
+}`}
+/>
+
+## Ingredients
+
+<Table>
+  <THead>
+    <Tr>
+      <Th>Component</Th>
+      <Th>Guidelines</Th>
+    </Tr>
+  </THead>
+  <TBody>
+    <Tr>
+      <Td>
+        <strong>Badge</strong>
+      </Td>
+      <Td>
+        Use a <Anchor href="/components/badge#decorative">decorative Badge</Anchor> to highlight a “recommended” plan and for a customer’s current plan or the status of that current plan.
+        We recommend a maximum of 2 badges total, to not overwhelm the users. Use the placeholder text to add more detail about the plan.
+      </Td>
+    </Tr>
+    <Tr>
+      <Td>
+        <strong>Heading</strong>
+      </Td>
+      <Td>
+        This is a smaller heading used for displaying the plan name or other details if including the plan name as the display heading.
+        Refer to the <Anchor href="/components/heading#accessibility">heading documentation</Anchor> for accessibility of headers.
+      </Td>
+    </Tr>
+    <Tr>
+      <Td>
+        <strong>Display Heading</strong>
+      </Td>
+      <Td>
+        Use this largest heading for displaying the price of the plan. If the price is not available, it can be used to display the plan name.
+      </Td>
+    </Tr>
+    <Tr>
+      <Td>
+        <strong>Paragraph subheader</strong>
+      </Td>
+      <Td>
+        The subheader can be placed above or below the display heading. Use this to add additional context to the price or plan.
+        For example “Included” or “Plus fees” or “Best for XYZ”
+      </Td>
+    </Tr>
+    <Tr>
+      <Td>
+        <strong><Anchor href="/components/button">Button</Anchor> (card variant)</strong>
+      </Td>
+      <Td>
+        Use a secondary button to select the plan. We suggest using the button text “Select plan”.
+        For the current plan, hide the button or show an CTA. For example, it could navigate to the feature the user has already selected.
```

Comment:
```suggestion
        Use a secondary button to select the plan using the text “Select plan”.
        For the current plan, hide the button or show a CTA. For example, it could navigate to the feature the user has already selected.
```